core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -249,23 +249,6 @@ public static long parseTimeSize(String timeSize) {
     }
   }
 
-  /**
-   * Parses a boolean from a string.
-   *
-   * @param bool the boolean to parse
-   * @return whether the value was "true" or "false"
-   * @throws IllegalArgumentException if the value cannot be parsed
-   */
-  public static boolean parseBoolean(String bool) throws IllegalArgumentException {
-    if (bool.equalsIgnoreCase("true")) {
```

Comment:
question: if in the `alluxio-site.properties` file, a user sets a property to be `True` or `FALSE` (allowed previously),
will the current implementation throw RTE?

Reply:
Do we have a test testing this scenario?

Reply:
`Boolean.parseBoolean` would return true matching true ignore cases and return false for all other values. The difference is here is whether you want to enforce `false` to only match false ignore cases. So all existing cases should still work, but "no" and "yes" will also not throw in the future.